src/contracts/decentralized_oracle.js

```diff
@@ -7,23 +7,23 @@ const qweb3 = new Qweb3(Config.QTUM_RPC_ADDRESS);
 export async function vote(args) {
   const { 
     contractAddress,
-    resultIndex, 
-    botAmount, 
+    index, 
+    amount, 
```

Comment:
I feel like we should call this something other than `amount` as there is already an `amount` field in the qtum-cli. What do you think?